plugins/node/instrumentation-fs/src/instrumentation.ts

```diff
@@ -253,6 +280,8 @@ export default class FsInstrumentation extends InstrumentationBase<FS> {
         return original.apply(this, args);
       }
     };
+    Object.assign(rv, original);
```

Comment:
If I understand correctly, this `assign` is required so that 1st-level functions like `realpath` will also have 2nd-level functions (`native`) copied to the patch function object, which is then pathed separately in a future iteration of the `wrap` function, right?

Can you please add a comment explaining the purpose of this line for future readers?

Reply:
You understand correctly.

Do you think it would make sense to create a wrapper function for Object.assign(rv, original) with a name like 'patchedFunctionWithOriginalProperties(function: F, original: O): O & F`, since Object.assign is used in two places and the comments could get desynchronized? I think then the function name speaks for itself without comments?

Reply:
I like the idea of extracting it into a function. 

The name `patchedFunctionWithOriginalProperties` describes the action it performs `Object.assign(rv, original)`, but not the intent. I think future readers would appreciate us stating it is important for 2-level functions like `realpath.native` to retain the 2nd-level when patching the 1st-level, then it is obvious why it is used and what function it serves in the code.